### ISSUE TITLE
Serialize a bean

### ISSUE THREAD

**Issue by marcteyssier** [thecodingmachine/database.tdbm#67]
Nice to have, add the option to serialize in php the bean object.
(Very interesting for the right plugin ;-) )

**Comment by moufmouf** [comment]
This raises a lot of issues that we need to discuss:
- What do we do with linked beans? With many-to-many relationships? Do we serialize them too? If yes, when do we stop (otherwise, there could be infinite loops)
- When we unserialize a bean, what happens if we save it? Is it possible? Must we instead "attach" the bean back to the TDBMService? If we attach the bean, what is happening? Do we update the data in database or do we fetch the data from database and update the data in the bean?

These questions must be answered before we start developing something!
